app/services/check_invoice_expire.rb

```diff
@@ -0,0 +1,13 @@
+class CheckInvoiceExpire < ApplicationService
+  def call
+    invoices.each do |invoice|
```

Comment:
```suggestion
    invoices.find_each do |invoice|
```
references: https://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each